src/routers/sessions.py

```
@@ -84,23 +84,14 @@ async def get_sessions(
+        if filter_param == {}:  # Explicitly check for empty dict
+            filter_param = None
 
-    return await paginate(
-        db,
-        await crud.get_sessions(
-            workspace_name=workspace_id,
-            is_active=is_active_param,
-            filter=filter_param,
-        ),
+    return await apaginate(
+        db, await crud.get_sessions(workspace_name=workspace_id, filter=filter_param)
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)